eclipse-collections-code-generator/src/main/resources/api/map/mutablePrimitiveObjectMap.stg

```diff
@@ -45,20 +53,71 @@ public interface Mutable<name>ObjectMap\<V> extends <name>ObjectMap\<V>, Mutable
     }
 
     /**
+     * Puts all of the key/value mappings from the specified map into this map. If this
+     * map already has a value associated with one of the keys in the map, it will be
+     * replaced with the value in {@code map}.
+     * @param map the map to copy into this map
      * @since 5.0.
      */
     void putAll(<name>ObjectMap\<? extends V> map);
 
+    /**
+     * Removes the mapping associated with the key, if one exists, from the map.
+     * @param key the key to remove
+     * @see #remove(<type>)
+     */
     V removeKey(<type> key);
 
+    /**
+     * Removes the mapping associated with the key, if one exists, from the map.
+     * @param key the key to remove
+     * @see #removeKey(<type>)
+     */
     V remove(<type> key);
 
+    /**
+     * Retrieves the value associated with the key if one exists; if it does not,
+     * associates a value with the key.
+     * a new value with they key
```

Comment:
This sentence fragment looks like it might be superfluous.

Reply:
Good catch! I'll fix it.